.github/CODEOWNERS

```diff
@@ -0,0 +1,34 @@
+# GitHub CODEOWNERS definition
+# See: https://help.github.com/articles/about-codeowners/
+
+* @elastic/beats
+
+/auditbeat/module/ @elastic/secops
+/x-pack/auditbeat/ @elastic/secops
+
+/packetbeat/protos/ @elastic/secops
+/x-pack/packetbeat/ @elastic/secops
+
+/metricbeat/module/ @elastic/infrastructure
+/metricbeat/modules.d/ @elastic/infrastructure
+/metricbeat/module/elasticsearch/ @elastic/stack-monitoring
+/metricbeat/module/kibana/ @elastic/stack-monitoring
+/metricbeat/module/logstash/ @elastic/stack-monitoring
+/x-pack/metricbeat/module/ @elastic/infrastructure
+/x-pack/metricbeat/modules.d/ @elastic/infrastructure
```

Comment:
this is "libbeat - Xpack Monitoring" as far as I know and should be owned by elastic/beats i believe. 
Same for `/x-pack/filebeat/module/` below.